python-threatexchange/threatexchange/exchanges/tests/test_state.py

```diff
@@ -414,12 +472,14 @@ def key(n):
         },
         # Hash 1 again
         {
-            md5(1): SimpleFetchedSignalMetadata(
+            md5(1): FakeSignalMetadata(
                 [
-                    SignalOpinion(
-                        1, SignalOpinionCategory.INVESTIGATION_SEED, {t2, t3}
+                    SignalOpinionWithOwner(
+                        1, SignalOpinionCategory.INVESTIGATION_SEED, {t2, t, 3}
```

Comment:
This changed significantly. Went from t2, t3 to t2, t, 3. 

Is it a typo?

Reply:
Hmm, this looks like a vscode cursor bug, but confused about why tests wouldn't fail